src/Apps/Collect2/Routes/Collection/index.tsx

```diff
@@ -77,7 +77,10 @@ export class CollectionApp extends Component<CollectionAppProps> {
     const showCollectionHubs = collection.linkedCollections.length > 0
 
     const socialImage =
-      headerImage || fallbackHeaderImage?.edges[0]?.node?.image?.resized.url
+      headerImage ||
+      (fallbackHeaderImage?.edges &&
+        fallbackHeaderImage?.edges[0]?.node?.image?.resized.url)
+
```

Comment:
I refactored this to more explicitly check for `null`. Regarding testing, I attempted to add tests but the CollectionApp component doesn't currently have any tests and it's quite a bit of work to set up so I created a [ticket](https://artsyproduct.atlassian.net/browse/FX-1849) to do so.

Reply:
@xtina-starr - I _think_ `edges[0]?` should work? Can try it out in the chrome console: 

<img width="270" alt="Screen Shot 2020-03-11 at 3 51 28 PM" src="https://user-images.githubusercontent.com/236943/76471580-41598080-63b0-11ea-9015-31c8d4004888.png">